cl/opinion_page/views.py

```diff
@@ -465,6 +487,32 @@ def view_recap_document(
 
     # Override the og:url if we're serving a request to an OG crawler bot
     og_file_path_override = f"/{rd.filepath_local}" if is_og_bot else None
+    if rd.attachment_number:
+        view_authorities_url = reverse(
+            "view_attachment_authorities",
+            args=[
+                rd.docket_entry.docket_id,
+                rd.document_number,
+                rd.attachment_number,
+                rd.docket_entry.docket.slug,
+            ],
+        )
+    else:
+        view_authorities_url = reverse(
+            "view_document_authorities",
+            args=[
+                rd.docket_entry.docket_id,
+                rd.document_number,
+                rd.docket_entry.docket.slug,
+            ],
+        )
```

Comment:
I think we can do this in the template, can't we? I think that's where we'd normally figure out the URL for something?

Reply:
> I think we can do this in the template, can't we?

You're right! We could move this code into a dedicated method in the `RECAPDocument` class, similar to the `get_absolute_url` method.